snuba/cli/rust_consumer.py

```diff
@@ -106,6 +106,12 @@
     default=None,
     help="Kafka group instance id. passing a value here will run kafka with static membership.",
 )
+@click.option(
+    "--python-max-queue-depth",
+    type=int,
+    default=None,
+    help="How many messages should be queued up before the Python message processor before backpressure kicks in. Defaults to the number of processes.",
```

Comment:
This reads wrong to me, but maybe I'm misunderstanding it?

Reply:
the default or which part?

Reply:
"queued up before the Python message processor before backpressure kicks in."

Reply:
I think both of those would be improvements